packages/client/webapp/src/views/Home.vue

```diff
@@ -1,53 +1,103 @@
 <script setup lang="ts">
-import Button from '@/components/Button.vue';
-import { uncloak } from '../utils/uncloak';
-
-async function apiCall(type = '') {
-	if (type === 'user') {
-		const data = await uncloak('/api/user/me');
-		if (data === null) {
-			console.log('Unauthorized Access');
-		}
-		console.log(`Response: ${JSON.stringify(data)}`);
-	}
-	if (type === 'admin') {
-		const data = await uncloak('/api/admin');
-		if (data === null) {
-			console.log('Unauthorized Access');
-		}
-		console.log(`Response: ${JSON.stringify(data)}`);
-	}
+import ProjectCard from '@/components/projects/ProjectCard.vue';
+import IconTime32 from '@carbon/icons-vue/es/time/32';
+import IconChevronLeft32 from '@carbon/icons-vue/es/chevron--left/32';
+import IconChevronRight32 from '@carbon/icons-vue/es/chevron--right/32';
+import IconFire32 from '@carbon/icons-vue/es/fire/32';
+import IconCoronavirus32 from '@carbon/icons-vue/es/coronavirus/32';
+import NewProjectCard from '../components/projects/NewProjectCard.vue';
+
+const enum Categories {
+	Recents = 'Recents',
+	Trending = 'Trending',
+	Epidemiology = 'Epidemiology'
 }
+
+const categories = new Map<string, { icon: string }>([
+	[Categories.Recents, { icon: IconTime32 }],
+	[Categories.Trending, { icon: IconFire32 }],
+	[Categories.Epidemiology, { icon: IconCoronavirus32 }]
+]);
+
+const dummyProjects: string[] = Array(8)
+	.fill('Project')
+	.map((project, index) => `${project} ${index}`);
 </script>
 
 <template>
 	<section>
-		<p>Test API calls/Home placeholder</p>
-		<div>
-			<Button>Normal</Button>
-			<Button action>Action</Button>
-			<Button danger>Danger</Button>
-			<Button info>Info</Button>
-		</div>
-		<div>
-			<Button @click="apiCall('user')">User Call</Button>
-			<Button @click="apiCall('admin')">Admin Call</Button>
-		</div>
+		<h2>Projects</h2>
+		<template v-for="[key, value] in categories" :key="key">
+			<h3><component :is="value.icon" />{{ key }}</h3>
```

Comment:
What does `<component :is="value.icon">` do?

Reply:
It grabs the appropriate icon from the `categories` map above.